pvc.yaml

```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: claim
+  annotations:
+    volume.beta.kubernetes.io/storage-class: rook-ceph-block-karan
```

Comment:
rook-ceph-block-karan -> please use jinja2 template (you can find examples in the templates folder and move it there)

Reply:
This file should be under templates folder

Reply:
@RazTamir  can you review the new patch. i rewrote it for better understanding and simplified it

Reply:
@ksandha where it is?

Reply:
https://github.com/red-hat-storage/ocs-ci/commit/eeee3295f8f07c334ba9465236b28b01fd262ace